Client.Linq/Internal/Expressions/String/StartsWith.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class StartsWith : IExpressionPart
```

Comment:
Please rename to `HasPrefix`
```suggestion
    internal class HasPrefix : IExpressionPart
```